grep-kong-version.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
This script is in pretty rough shape, but I get that this is not what we're trying to fix at the moment. I'm suggesting a couple small improvements:

1. use `$(command)` instead of backticks
2. quote usage of `$KONG_SOURCE_LOCATION`